integration-intygstjanst/src/main/java/se/inera/intyg/minaintyg/integration/intygstjanst/client/SendCertificateUsingIntygstjanstService.java

```diff
@@ -0,0 +1,48 @@
+package se.inera.intyg.minaintyg.integration.intygstjanst.client;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import se.inera.intyg.minaintyg.integration.api.certificate.SendCertificateIntegrationRequest;
+
+@Service
+public class SendCertificateUsingIntygstjanstService {
+
+  private final WebClient webClient;
+  private final String scheme;
+  private final String baseUrl;
+  private final int port;
+  private final String endpoint;
+
+  public SendCertificateUsingIntygstjanstService(
+      @Qualifier(value = "intygstjanstWebClient") WebClient webClient,
+      @Value("${integration.intygstjanst.scheme}") String scheme,
+      @Value("${integration.intygstjanst.baseurl}") String baseUrl,
+      @Value("${integration.intygstjanst.port}") int port,
+      @Value("${integration.intygstjanst.certificate.endpoint}") String endpoint) {
```

Comment:
I suggest that you add a new property that includes the /send part of the url. Maybe something like: integration.intygstjanst.sendcertificate.endpoint